src/Compilers/VisualBasic/Portable/Symbols/Symbol.vb

```diff
@@ -481,7 +481,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
 
         ''' <summary>
-        ''' Returns data decoded from Obsolete attribute or null if there is no Obsolete attribute.
+        ''' Returns data decoded from Obsolete/Experimental attribute or null if there is no Obsolete/Experimental attribute.
```

Comment:
Can you update `Symbol.cs`, similarly?

https://github.com/dotnet/roslyn/blob/294cbb64e624f9783821965bfeda232870a97925/src/Compilers/CSharp/Portable/Symbols/Symbol.cs#L1343-L1347